apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-menu/index.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+import { LAUNCH_STORE } from '../stores';
+import { LaunchStep, LaunchSequence } from '../../../common/data-stores/launch/data';
+import LaunchMenuItem from './item';
+import './styles.scss';
+
+const LaunchStepMenuItemTitles = {
+	[ LaunchStep.Name ]: __( 'Name your site', 'full-site-editing' ),
+	[ LaunchStep.Domain ]: __( 'Select a domain', 'full-site-editing' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Select a domain' )` *ES Score: 6*